packages/frameworks/package.json

```diff
@@ -1,14 +1,22 @@
 {
   "name": "@vercel/frameworks",
   "version": "0.2.0",
-  "main": "frameworks.json",
+  "main": "./dist/frameworks.js",
+  "types": "./dist/frameworks.d.ts",
   "license": "UNLICENSED",
```

Comment:
You should add `"files": [ "dist" ]` to avoid publishing unnecessary files to npm